dpctl-capi/include/dpctl_sycl_device_interface.h

```diff
@@ -282,7 +259,18 @@ bool DPCTLDevice_IsHostUnifiedMemory(
  * @return   True if the underlying sycl::device are same, false otherwise.
  */
 DPCTL_API
-bool DPCTLDevice_AreEq(__dpctl_keep const DPCTLSyclDeviceRef DRef1,
-                       __dpctl_keep const DPCTLSyclDeviceRef DRef2);
+bool DPCTLDevice_AreEq(__dpctl_keep const DPCTLSyclDeviceRef DevRef1,
+                       __dpctl_keep const DPCTLSyclDeviceRef DevRef2);
+
+/*!
+ * @brief Checks if device has aspect.
+ *
+ * @param    DRef       Opaque pointer to a sycl::device
+ * @param    AT         DPCTLSyclAspectType of device::aspect.
+ * @return   True if sycl::device has device::aspect, else false.
+ */
+DPCTL_API
+bool DPCTLDevice_HasAspect(__dpctl_keep const DPCTLSyclDeviceRef DRef,
+                           __dpctl_keep const DPCTLSyclAspectType AT);
```

Comment:
`DPCTLSyclAspectType` is not a pointer. `__dpctl_keep const` indicate that the function does not delete or change the pointer. It is not needed for enum types.